components/dashboard/src/components/podkit/buttons/LinkButton.tsx

```diff
@@ -19,9 +18,11 @@ export interface LinkButtonProps extends ButtonProps {
 export const LinkButton = React.forwardRef<HTMLButtonElement, LinkButtonProps>(
     ({ asChild, children, href, ...props }, ref) => {
         return (
-            <Button ref={ref} {...props} asChild>
-                <Link to={href}>{children}</Link>
-            </Button>
+            <Link to={href}>
```

Comment:
This unfortunately causes a double focus target in Chrome. Is there a way for playwright to find a link instead?
<img width="408" alt="image" src="https://github.com/gitpod-io/gitpod/assets/29888641/85b2c72b-025b-4b56-9e33-5544afe0d675">

Reply:
@filiptronicek  We have two `New Workspace` button in this page, their old struct is like
- When there's no workspaces -> `LinkButton` (inside is `Button>Link`)
- With workspaces -> `Link>Button`

Change the second button into LinkButton then find with role `link` works in playwright too. But we do want them to  be button no?

Reply:
Or since we tested them alright, we could revert this changes, since it's not related to issue we want to fix

Reply:
I'll do revert podkit changes, since we don't play with playwright a lot

Reply:
reverted @filiptronicek